src/template-tags/tickets.php

```diff
@@ -445,7 +445,7 @@ function tribe_tickets_get_ticket_stock_message( Tribe__Tickets__Ticket_Object $
 
 		$sold_label = __( 'issued', 'event-tickets' );
 		if ( 'Tribe__Tickets__RSVP' === $ticket->provider_class ) {
-			$sold_label = _x( 'RSVP\'d Going', 'separate going and remain RSVPs', 'event-tickets' );
+			$sold_label = _x( "RSVP'd Going", 'separate going and remain RSVPs', 'event-tickets' );
```

Comment:
I feel bad about the translation thrash, but then again I feel good about not passing the `\` through to the translators?

Reply:
👍